scripts/publish-npm-packages.sh

```diff
@@ -44,3 +44,17 @@ echo "$registry_without_protocol/:_authToken=${NPM_TOKEN}" >> ~/.npmrc
 for file in ./npm-artifacts/*.tgz; do
     npm publish "$file" --tag "$dist_tag" --registry "$registry"
 done
+
+# Check if any files in packages/grafana-e2e-selectors were changed. If so, add a 'modified' tag to the package
+COUNT=$(git diff HEAD~1..HEAD --name-only -- packages/grafana-e2e-selectors | awk 'END{print NR}')
```

Comment:
Nit: maybe `CHANGES_COUNT` would be slightly better name, not too important though.